### PR TITLE
Messaging update 2: messaging harder!

### PR DESCRIPTION
**Pull request by skizotenigma**
Redid all the previous edit and now a completely revamped generic_api.py!

### CODE REVIEW

File: src/api/discord_embeds.py [hunk 1]
```diff
@@ -21,14 +21,24 @@ def send_item(self, item: str, image:  np.ndarray, location: str):
...
+        else:                 
+            e = discord.Embed(
+                title="Item Stashed!",
+                description=f"{item} at {location}",
+                color=self._get_Item_Color( item),
+            )
         e.set_thumbnail(url=f"{self._psnURL}41L6bd712.png")
-        e.set_image(url=f"attachment://{imgName}.png")
+        if item == "magic_gg_club":
+            e.set_image(url=f"https://i.insider.com/5bd6fce948eb12606c6c4112?width=700&format=jpeg")
```
Comment (FrancescoZini) [inline src/api/discord_embeds.py:39]: lol

Comment (skizotenigma) [reply src/api/discord_embeds.py:39]: SHHHH

**Comment by mgleed** [comment]
Just reviewing some old PR's @skizotenigma , think this one is salvageable? Will obviously need to merge master. What does it do? Lmk.

**Comment by mgleed** [comment]
Closing, attempting to merge into #500

**Pull request closed without merging**